app/src/main/java/com/github/houseorganizer/houseorganizer/panels/MainScreenActivity.java

```diff
@@ -284,4 +285,13 @@ public void rotateLists(View view) {
                 break;
         }
     }
+
+    public void goToOfflineScreen(View v) {
```

Comment:
Is this method executed when the user suddenly loses connection?

Reply:
For now it's executed when the `Go offline` button is pressed. Depending on how much time I have left, I will change it so that when the user loses connection they go to offline mode instead.